frontend/src/scenes/web-analytics/webAnalyticsLogic.tsx

```diff
@@ -1082,6 +1087,7 @@ export const webAnalyticsLogic = kea<webAnalyticsLogicType>([
                               },
                           }
                         : null,
+                    // TODO: What to do with this one if conversion goal is set? Hide it? Disable it?
```

Comment:
Any ideas @robbie-c? I believe we should simply hide the parts of the UI that aren't "filtered" by conversion goals when this happens.

Reply:
Yeah good Q.

One idea could be that it works like list of funnels. E.g. if you have Action G as the goal, and Actions A B C as other actions, it could shows

Action A - some conversion stats from A -> G
Action B - some conversion stats from B -> G
Action C - some conversion stats from C -> G

though maybe it should be the other way around? e.g. G -> A

I think it's also fine to just hide this tile when a conversion goal is set :)

Reply:
Decision for now: just keep it hidden, but this would be a cool visualization :)